web/src/util/serialization/activities.test.js

```diff
@@ -600,6 +605,7 @@ describe('APD activity serializer', () => {
             id: 'eugene',
             title: 'Boss',
             description: 'friend of the show',
+            keyPersonnel: false,
```

Comment:
should this be `isKeyPersonnel`?

Reply:
on the backend right now, this field is called `keyPersonnel`

Reply:
Oh, whoops.  Should have pulled the code to look at this in context.  😬